save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/cosv/RawCosvExt.kt

```diff
@@ -0,0 +1,20 @@
+package com.saveourtool.save.entities.cosv
+
+import com.saveourtool.save.entities.vulnerability.VulnerabilityDateDto
+import com.saveourtool.save.info.UserInfo
+import com.saveourtool.osv4k.RawOsvSchema as RawCosvSchema
+
+/**
+ * @property metadata
+ * @property rawContent
+ * @property saveContributors save's user from [com.saveourtool.osv4k.OsvSchema.contributors]
+ * @property tags
+ * @property timeline
+ **/
+data class RawCosvExt(
```

Comment:
let's rename it to `VulnerabilityNew`. It will replace `Vulnerability` and `VulnerabilityDto` in future

Reply:
will do in next PR